### ISSUE TITLE
PL 117-68 - Bill S.1095 - In-State Tuition Updates

### ISSUE THREAD

**Issue by mrblanco1** [department-of-veterans-affairs/va.gov-team#36489]
## Issue Description

PL 117-68 - Bill S.1095, now PL 117-68, and I am attaching a summary/views along with slides (relevant slide is #3).  v

The requirement is bringing the DEA benefit (chapter 35) calculator to include in-state tuition at public IHLs with an effective date of implementation date of August 1, 2022.  Using the screenshot below, the calculator for this benefit would show the “GI Bill Pays to school” calculating the same rate as a 100% Post-9/11 veteran, $10,145, instead of $0/yr.  And out of pocket tuition as $0, as it does for a 100% Post-9/11 Veteran.

Link to the full bill is here: https://www.congress.gov/117/plaws/publ68/PLAW-117publ68.pdf

![image.png](https://images.zenhubusercontent.com/60e78c6b02152f0a04637267/686cd7ca-abfe-4b52-bdca-aeeacbc0a298)

---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by dan-brooking** [comment]
@mrblanco1 Is the goal here to get Chapter 35 (DEA) benefits to calculate the same as Chapter 33 (Post 9/11)? 

The reason I ask is because the ticket description says "The requirement is bringing the DEA benefit (chapter 35) calculator to include in-state tuition.....calculating the same rate as a 100% Post-9/11 veteran, $10,145" but in testing, it looks like the post 9/11 "GI Bill Pays" is greater than $10,145 for the few schools I looked at.

I'm not clear on what we should do with the $10,145 value as it appears to contradict what we see with the post 9/11 chapter 33 calculations. I've attached a couple screenshots showing what I am seeing.

![image](https://user-images.githubusercontent.com/90346049/154303293-3ef38e62-2516-4106-b1c5-8bf5b9a8b809.png)
![image](https://user-images.githubusercontent.com/90346049/154303121-3beb4373-c0be-4347-87db-15f601f8fc85.png)

I'm wondering if the $10,145 was mentioned in the ticket because that was relevant to the specific screenshot shared as an example?
